python/paddle/distributed/__init__.py

```diff
@@ -24,10 +24,11 @@
 
 from . import collective
 from .collective import *
+from ..fluid.dygraph.parallel import prepare_context  #DEFINE_ALIAS
```

Comment:
verify needed

Reply:
prepare_context should access by paddle.distributed.prepare_context in 2.0API